app.js

```diff
@@ -1,18 +1,23 @@
-const express = require('express');
+const express = require("express");
 const app = express();
+const endPoints = require("./endpoints.json")
 
-const{ getTopics } = require('./controllers/topics.controller')
+const { getTopics } = require("./controllers/topics.controller");
 
-app.get('/api/topics', getTopics);
+app.get("/api/topics", getTopics);
 
+app.get("/api", (req, res) => {
+    console.log(endPoints)
+    console.log({endPoints},"<<<<")
+    res.status(200).send({endPoints})
```

Comment:
Good key name for the response, well done.

Reply:
I know the logic is quite simple for this endpoint, but we still want to keep consistency with our MVC design architecture, so I would advise splitting this into a controller function just like you did with topics, and send the response in that controller rather than trying do it all in app.js - that way we separate out the concerns accordingly.